examples/co2_analysis.py

```diff
@@ -3,76 +3,97 @@
 of the same well test, a tracer concentration can be determined.
 """
 
-from pathlib import Path
-
 import cv2
 import matplotlib.pyplot as plt
+import numpy as np
 import skimage
 
 import daria
 
 
-def preprocessing(img):
-    """Standard curvature and color correction. Return ROI."""
+def preprocessing(img: np.ndarray) -> np.ndarray:
+    """Standard for reading an image from file, applying curvature correction
+    and and color correction.
 
-    # Curvature correctio
-    img = daria.curvature_correction(img)
+    Args:
+        img (np.ndarray): image array
 
-    # Preprocessing. Transform to RGB space
-    img = cv2.cvtColor(img, cv2.COLOR_BGR2RGB)
+    Returns:
+        np.ndarray: corrected image
+    """
+
+    # Define curvature correction object, initiated with config file
+    # (which can be created the workflow presented in the Jupyter notebook
+    # examples/notebooks/curvature_correction_walkthrough.ipynb).
+    curvature_correction = daria.CurvatureCorrection(
+        config_source="./images/config.json", width=2.8, height=1.5
+    )
 
-    plt.imshow(img)
-    plt.show()
+    # Apply curvature correction.
+    img = curvature_correction(img)
 
-    # Color correction
+    # Transform to RGB space.
+    img = cv2.cvtColor(img, cv2.COLOR_BGR2RGB)
```

Comment:
I'm thinking that there should be a way to hide the change in color space within the colorcorrection. Perhaps by first creating a daria image, and then use the colorcorrection in the daria image and make use of the colorspace attribute?